### DISCUSSION TITLE
Using `python -m pudb ...` , how can I automatically continue and quit?

### DISCUSSION THREAD

**Discussion by cgebbe** [inducer/pudb#651] [Q&A]
When I call `python -m pudb myscript.py`, 
then a screen pops at the beginning where I continue
and upon successful completion another screen pops up where I quit.

Can I prevent these two screen pop-ups and only enter PUDB upon uncaught exceptions?

**Comment by mvanderkamp** [comment]
Does the `-c` / `--continue` option satisfy your needs? e.g.
```bash
python -m pudb --continue myscript.py
```

**Comment by cgebbe** [comment]
It solves the continue problem, but I'd also like to skip the quit screen upon successful script completion.

**Comment by mvanderkamp** [comment]
For that you can press `<ctrl-p>` in the debugger to open the settings and uncheck the "Prompt before quitting option":
```
General:                                            
                                                    
[X] Show Line Numbers                               
[ ] Prompt before quitting                          
[ ] Hide command line(ctrl x) window when not in use
```

**Comment by cgebbe** [comment]
Awesome, didn't notice this option beforehand!
